integrations/node/workload-executor.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+const MongoClient = require('../../node-mongodb-native').MongoClient;
+const fs = require('fs');
+const assert = require('assert');
+const omit = require('lodash.omit');
+
+let interrupted = false;
+process.on('SIGINT', () => (interrupted = true));
+
+const executors = {
+  find: (collection, args) =>
+    collection.find(args.filter || {}, omit(args, ['filter'])).toArray(),
+
+  insertOne: (collection, args) =>
+    collection.insertOne(args.document, { forceServerObjectId: true }),
```

Comment:
Note - I get a bunch of duplicate key errors without `forceServerObjectId: true`.

Reply:
can you paste the errors here? That's very strange, I wouldn't expect that to happen.

Reply:
I removed the `forceServerObjectId` and I'm no longer getting the errors in my evergreen runs; not sure what was going on there, but I'll update if they pop up again.